sqlite.cpp

```diff
@@ -97,29 +290,22 @@ sqlite3_stmt *SQLite::prepare(const char *query) {
 	// Get database pointer
 	sqlite3 *dbs = get_handler();
 
-	if (!dbs) {
-		print_error("Cannot prepare query! Database is not opened.");
-		return nullptr;
-	}
+	ERR_FAIL_COND_V_MSG(dbs == nullptr, nullptr, "Cannot prepare query! Database is not opened.");
 
 	// Prepare the statement
-	sqlite3_stmt *stmt;
+	sqlite3_stmt *stmt = nullptr;
 	int result = sqlite3_prepare_v2(dbs, query, -1, &stmt, nullptr);
 
 	// Cannot prepare query!
-	if (result != SQLITE_OK) {
-		print_error("SQL Error: " + String(sqlite3_errmsg(dbs)));
-		return nullptr;
-	}
-
+	ERR_FAIL_COND_V_MSG(result != SQLITE_OK, nullptr, "SQL Error: " + get_last_error_message());
 	return stmt;
 }
 
 bool SQLite::bind_args(sqlite3_stmt *stmt, Array args) {
 	// Check parameter count
 	int param_count = sqlite3_bind_parameter_count(stmt);
 	if (param_count != args.size()) {
-		print_error("Query failed; expected " + itos(param_count) + " arguments, got " + itos(args.size()));
+		print_error("SQLiteQuery failed; expected " + itos(param_count) + " arguments, got " + itos(args.size()));
```

Comment:
Review with peers mentioned that the error message is not "SQLiteQuery" specific, so it should be just "Query".

Reply:
Ops, I've used the rename, and everything got renamed. I'll fix that.